### PR TITLE
adding `rgb_to_y`

### PR DESCRIPTION
**Pull request by nitaifingerhut**
#### Changes
adding a method for `rgb_to_y`. In many cases, only the luma channel is used in CV tasks, and this method calculates it without un-necessary compute. 

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🔬 New feature (non-breaking change which adds functionality)

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: kornia/color/ycbcr.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
 import torch
 import torch.nn as nn
 
+from torch import Tensor
```
Comment (nitaifingerhut) [inline kornia/color/ycbcr.py:4]: replaced all `torch.Tensor` with `Tensor`

File: test/color/test_ycbcr.py [hunk 2]
```diff
@@ -17,6 +17,13 @@ def test_cardinality(self, device, dtype, shape):
         img = torch.ones(shape, device=device, dtype=dtype)
         assert kornia.color.rgb_to_ycbcr(img).shape == shape
 
+    @pytest.mark.parametrize("shape", [(3, 4, 4), (2, 3, 4, 4), (2, 2, 3, 4, 4)])
```
Comment (shijianjian) [inline test/color/test_ycbcr.py:20]: Should 22344 success or fail? It's not an image with 5 dims, right?

**Review by shijianjian — APPROVED** [review]
LGTM

File: test/color/test_ycbcr.py (hunk 2 above)
Comment (nitaifingerhut) [reply test/color/test_ycbcr.py:20]: success. 5 dims is for volumetric tensors (say a video of `BTCHW`). basically this method only multiplies the channels by scalars, so it would work on any tensor with shape `(*, C, H, W)`.

Comment (shijianjian) [reply test/color/test_ycbcr.py:20]: For volumetric data, it shall be `BCDHW`, so that the depth is in the third axis. I reckon we shall only support image types at the moment.

Comment (nitaifingerhut) [reply test/color/test_ycbcr.py:20]: ok, so it will work
do you want me to remove the check for 5d inputs?

Comment (nitaifingerhut) [reply test/color/test_ycbcr.py]: @shijianjian fixed

**Review by edgarriba — APPROVED** [review]
LGTM. Add in a follow up pr the function to docs
